src/transactionsFactories/smartContractTransactionsFactory.spec.ts

```diff
@@ -1,15 +1,15 @@
+import BigNumber from "bignumber.js";
```

Comment:
I don't think this is used anymore. Can be deleted if so.

Reply:
Still needed on `smartContractFactory.createTransactionForExecute`, which is not yet adjusted to use `bigint` (to be done in another PR).